### ISSUE TITLE
Text size is rendering incorrectly

### ISSUE THREAD

**Issue by llewellynathan** [benjamindenboer/FramerInput#12]
Everything was working fine with the module but all of a sudden the text size starting rending in the preview at about half the size that it should be.

**Comment by llewellynathan** [comment]
This appears to be related to the device I choose. Text size renders correctly/as expected when previewing on a Macbook Air but renders at half the declared size when I switch to Canvas or Macbook Pro.

**Comment by llewellynathan** [comment]
When I preview on the web the text size appears at about half the declared size no matter what device I have selected in Framer.

**Comment by kyleartbow** [comment]
Yes, me too. My field is located in the Side menu bar. Here is a link: https://framer.cloud/pHTNb

**Comment by bruskowski** [comment]
I had a similar issue (size was twice as big as it should have been when using a custom device) and could resolve it by editing lines 177 and 226 to
`dpr = 2 # @_checkDevicePixelRatio()`

Obviously that's just a hack, fixing the function _checkDevicePixelRatio() would make more sense for sure.
